### ISSUE TITLE
Alternative completed guide on OS development

### ISSUE THREAD

**Issue by iKlsR** [SamyPesse/How-to-Make-a-Computer-Operating-System#120]
Every few months this pops up on HN or reddit and most people are genuinely interested in the topic but it looks like this project is on an extended pause. An alternate and "complete" in a sense read is http://littleosbook.github.io/.

There is also https://github.com/tuhdo/os01

**Comment by fylux** [comment]
Is this project going to be continued or is already done?

**Comment by aswinmohanme** [comment]
Yeah , I think it's pretty much Dead. But meanwhile you could checkout [Broken Thorn's Kernel Development Serires](http://www.brokenthorn.com/Resources/OSDevIndex.html)

**Comment by fylux** [comment]
@aswinmohanme Isn't that project dead too?

**Comment by urielsalis** [comment]
It is, and it does some things that are broken/are bad. There is a section is OSDev with the common errors
